README.rst

```diff
@@ -110,21 +249,21 @@ The code in this repository is licensed under the GPL v3.0 unless otherwise note
 How To Contribute
 #################
 
-To contribute, please make a pull request in this repositry on Github: `Open edX Caliper Tracking <https://github.com/ucsd-ets/openedx-caliper-tracking>`_. If you have any question or issue, please feel free to open an issue on Github: `Open edX Caliper Tracking <https://github.com/ucsd-ets/openedx-caliper-tracking>`_.
+To contribute, please make a pull request in the repository. If you have any questions or issues, please feel free to open an issue on Github: `Open edX Caliper Tracking <https://github.com/ucsd-ets/openedx-caliper-tracking/issues/new>`_.
```

Comment:
In which repository? Link of the repository?

Reply:
done.